### PR TITLE
Documentation changes in Normal Gamma and JointRV

### PR DESCRIPTION
**Pull request by mijo2**
#### References to other Issues or PRs

Closes #15063

#### Brief description of what is fixed or changed

--> Updated the documentation in two functions, NormalGamma and JointRV in stats/joint_rv_types.py

#### Release Notes

* stats
  * Updated some documentation in NormalGamma and JointRV functions

### CODE REVIEW

File: sympy/stats/joint_rv_types.py [hunk 1]
```diff
@@ -251,12 +254,25 @@ def NormalGamma(syms, mu, lamda, alpha, beta):
     Parameters
     ==========
 
-    syms: list/tuple/set of two symbols for identifying each component
+    syms: A symbol/str for identifying the random variable.
```
Comment (asmeurer) [inline sympy/stats/joint_rv_types.py]: Was this just wrong before? If it's really only supposed to be one symbol the parameter should be called `sym`.

Comment (mijo2) [reply sympy/stats/joint_rv_types.py]: I think it was wrong before as when going into function definitions of ```multivariate_rv```, it is clearly mentioned that it takes only **a** symbol, not multiple ones. I will update ```syms``` to ```sym```.

**Comment by oscarbenjamin** [comment]
If this is based on #15063 then it is important to keep the author attribution correct. For that you would need to start your commits from the other PR.

**Comment by mijo2** [comment]
@oscarbenjamin so I should push my commits to #15063 right?

**Comment by oscarbenjamin** [comment]
> @oscarbenjamin so I should push my commits to #15063 right?

The normal way to revive an old PR is to rebase it to current master and then add your own commits:
```
# Make your own branch:
git remote add othername https://github.com/othername
git fetch
git checkout othername/otherbranch
git checkout -b mybranch
# Assuming that upstream is sympy/sympy.git
git fetch upstream
git rebase upstream/master
#### Make your own changes, run tests, commit then
git push -u origin mybranch
#### Open a pull request
```

**Comment by mijo2** [comment]
@oscarbenjamin I revived the PR as you asked in a new PR. #18589

**Pull request closed without merging**